motioneye/diskctl.py

```diff
@@ -68,10 +108,20 @@ def _list_mounts():
 def _list_disks():
     if os.path.exists('/dev/disk/by-id/'):
         return _list_disks_dev_by_id()
-    
-    else:  # fall back to fdisk -l
+    if os.path.exists('/proc/mounts'):
         return _list_disks_fdisk()
+    else:  # fall back to gpart command - Linux should never get here
+        return _list_disks_gpart()
+
+def _list_disks_gpart():
+    try:
+        output = subprocess.check_output(['gpart', 'status'], stderr=utils.DEV_NULL)
+    except Exception as e:
+        logging.error('failed to list disks using "gpart status": %s' % e, exc_info=True)
 
+    disks = []
+
+    return disks
```

Comment:
I think here is something wrong since you return an empty array? Isn't it intended to return `output` and make it an empty array only on `except`?